WooCommerce/Classes/ViewRelated/Products/Edit Product/Edit External Link/ProductExternalLinkViewController.swift

```diff
@@ -122,24 +127,29 @@ private extension ProductExternalLinkViewController {
     }
 
     func configureExternalURL(cell: TitleAndTextFieldTableViewCell) {
-        let title = NSLocalizedString("Product URL", comment: "Title of the text field for editing the external URL for an external/affiliate product")
-        let placeholder = NSLocalizedString("Enter URL",
-                                            comment: "Placeholder of the text field for editing the external URL for an external/affiliate product")
-        let viewModel = TitleAndTextFieldTableViewCell.ViewModel(title: title,
+        let title = Strings.productURLTitle
+        let placeholder = Strings.enterURLPlaceholder
+        var viewModel = TitleAndTextFieldTableViewCell.ViewModel(title: title,
                                                                  text: externalURL,
                                                                  placeholder: placeholder,
                                                                  keyboardType: .URL,
                                                                  textFieldAlignment: .trailing,
                                                                  onTextChange: { [weak self] text in
                                                                     self?.externalURL = text
+                                                                    if text?.isValidURL() == true {
```

Comment:
I think we might want to consider empty string a valid case so that we don't block the user from saving an empty URL (which is allowed in core)?

Reply:
Good catch! Updated here 347a76b49f073ce44476c5bd89310ec57c848b58